src/.env.sample

```diff
@@ -11,3 +11,8 @@ RAILS_LOG_TO_STDOUT=true
 RAILS_ENV=
 SECRET_KEY_BASE=
 SENTRY_DSN=
+
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=root
+POSTGRES_HOST=db
+POSTGRES_PORT=5432
```

Comment:
[R] we'll need to remove all the values from the .env.sample for be consistent with the rest of the file